eZ/Publish/API/Repository/Values/User/Role.php

```diff
@@ -38,6 +48,41 @@
     protected $identifier;
 
     /**
+     * The status of the role.
+     *
+     * @var int One of Role::STATUS_DEFINED|Role::STATUS_DRAFT
+     */
+    protected $status;
+
+    /**
+     * Creation date of the role.
+     *
+     * @var \DateTime
+     */
+    protected $creationDate;
```

Comment:
This and `$modificationDate`, `$creatorId` and `$modifierId` below are not used and will not be used in 6.0 (we would need to change the DB for that), so I would rather remove them for the time being.